wagtailio/core/blocks.py

```diff
@@ -226,10 +231,24 @@ def get_context(self, value, parent_context=None):
             context["value"]["url"] = value["cta_page"].get_url
         return context
 
+
+class CTALinkStructValue(blocks.StructValue):
```

Comment:
Suggestion: We can take this one step further, if we move this class above and then assign it to the mixin just like how you've done in the CTABlock model, we can inherit this assignment from the mixin itself, we won't need to re-define it within each model that uses the Mixin. What do you think?

Reply:
Sounds good! :+1:

Reply:
implemented in 9d75fc4